File: K.K.Sir/Leap_Year.c
#include<stdio.h>
int IsLeap(int z);
int main()
{
	int m,j,y,df=0;
	printf("Enter a date(dd/mm/yy) : ");
	scanf("%d/%d/%d",&d,&m,&y);
	
	/*if(IsLeap(z)==1)
		{
			printf("%d is a leap year. \n",z);
		}
	else
		printf("\a %d is not a leap year. \n",z);
	*/
	if(m==2)
		{
			if(IsLeap(z)==1)	
		}
	return 0;
}
int IsLeap(int z)
{
	int f=0;
	if(y<1753)
		{
			if(y%4==0)
				f=1;
		}
	else
		{
			if(y%100==0)
				{
					if(y%400==0)
						f=1;
				}
			else
				{
					if(y%4==0)
						f=1;
				}
		}
	return f;
}

File: Java/LeapYear.java
import java.io.*;
public class LeapYear{
	public static void main(String[] args)throws IOException{
		InputStreamReader ir=new InputStreamReader(System.in);
		//System.out.print("\n Enter is no:");
		BufferedReader br=new BufferedReader(ir);
		String str=br.readLine();
			int year=Integer.parseInt(str);
			if(((year%4==0)&&(year%100!=0))||(year%400==0))
			{
				System.out.println("Leap year");
			}
			else
			{
				System.out.println("Not leap year");
			}
		}
}